code/modules/random_map/drop/droppod.dm

```diff
@@ -19,9 +25,9 @@
 	var/auto_open_doors
 
 	var/placement_explosion_dev =   1
-	var/placement_explosion_heavy = 2
-	var/placement_explosion_light = 6
-	var/placement_explosion_flash = 4
+	var/placement_explosion_heavy = 1
+	var/placement_explosion_light = 3
+	var/placement_explosion_flash = 2
```

Comment:
whats the point of having heavy explosion if it has same radius as dev?

Reply:
lol, lmao even. dunno how explosion values work so i just reduced them all by half